inception/inception-api-annotation-experimental/src/main/java/de/tudarmstadt/ukp/inception/experimental/api/messages/request/arc/SelectArcRequest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to the Technische Universität Darmstadt under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The Technische Universität Darmstadt
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.experimental.api.messages.request.arc;
+
+import java.util.List;
+
+import de.tudarmstadt.ukp.clarin.webanno.api.annotation.model.VID;
+import de.tudarmstadt.ukp.inception.experimental.api.model.FeatureX;
+
+public class SelectArcRequest
+{
+    private String clientName;
+    private long projectId;
+    private long documentId;
+    private VID arcId;
+    private List<FeatureX> features;
```

Comment:
Hm yes thats right, thanks. These Messages can now be discarded. It was intially made because the Experience API didnt contained the data and it was good for presentation purposes. The ExperienceAPI in its current version now has all the data required